docs/session-inspector/gestures.md

```diff
@@ -14,13 +14,20 @@ This can be done by pressing the `+` button in the bottom left, which will open
 [Gesture Builder](#gesture-builder) screen.
 
 Once a gesture has been created and saved, clicking its entry in the saved gestures list will show
-its expected actions as an overlay over the [screenshot](./screenshot.md). There are also 3 buttons
+its expected actions as an overlay over the [screenshot](./screenshot.md). There are also 4 buttons
 for each gesture:
 
 -   Execute the gesture
 -   Edit the gesture (will open it in the [Gesture Builder](#gesture-builder))
+-   Download the gesture as json file
 -   Delete the gesture
 
+## Upload Gestures from file
```

Comment:
Not sure if a dedicated section is needed here. I think it's fine to just expand lines 12-14 to also mention the upload functionality. It could be turned into a bulleted list with 2 items (creating a gesture with the + button, and uploading a gesture with the upload button)

Reply:
Agree. I have accommodated the details inside the create gesture section itself.